ldap-client.go

```diff
@@ -22,7 +22,19 @@ var (
 	// errUserNotIdentified represents a situation in which the user could not be identified.
 	errUserNotIdentified = fmt.Errorf("user could not be indentifed")
 )
-
+type LDAPClienter interface{
+	Connect() error
+	Bind(dn,password string) error
+	Close()
+	UsersSearch(orFilter string,uidAttr string) (bool, map[string]map[string][]string, error)
+	RunQueries(username string, queries []string) (results map[string]bool, err error)
+	GetAllGroupsByName(groupName string) ([]*LdapGroup, error)
+	GetAllGroupsWithMembersByDN(groupDN []string) ([]*LdapGroup, error)
+	ChangeADUserPassword(username, oldPassword, newPassword string) (err error)
+    ChangeOpenLDAPUserPassword(username, oldPassword, newPassword string) (err error)
```

Comment:
fix alignment (or is it a github issue?)

Reply:
github issue, I see it ok